etc/rc1

```diff
@@ -58,9 +58,10 @@ if test $RANK -eq 0 -a "${FLUX_SCHED_MODULE}" != "none" \
     flux module load ${FLUX_SCHED_MODULE:-sched-simple}
 fi
 
-test $RANK -ne 0 -o "${FLUX_INSTANCE_RESTART}" = "t" \
```

Comment:
typo in commit message for c5b89ed: `it's` should be `its`

Reply:
Also :+1: on this change!

Reply:
Pushed a fix for that and a dictionary addition for el7.

Also, just wanted to note that a KVS checkpoint is forced out after the start event but not the finish event, so the finish event won't likely be commited to the backing store until the kvs module is unloaded.